public/index.html

```diff
@@ -3,13 +3,17 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie</title>
-    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet"  href="style.css">
```

Comment:
<link rel="stylesheet"  href="style.css"> -> <link rel="stylesheet" href="style.css">  (remove double space betwen rel and href)